flexget/plugins/internal/api_bluray.py

```diff
@@ -126,7 +126,11 @@ def __init__(self, title, year):
             # Used for parsing some more data, sadly with soup
             self.url = result['url']
 
-            movie_info_response = requests.get(self.url).content
+            try:
+                movie_info_response = requests.get(self.url).content
+            except (requests.RequestException, ConnectionError) as e:
+                raise LookupError('Couldn\'t connect to blu-ray.com. %s' % self.url)
+                break
```

Comment:
i'm not sure what this is supposed to do

Reply:
Oops, leftover from the rough local debug. Gone now